src/orders/components/OrderChangeWarehouseDialog/OrderChangeWarehouseDialog.tsx

```diff
@@ -184,12 +194,19 @@ export const OrderChangeWarehouseDialog: React.FC<OrderChangeWarehouseDialogProp
             })}
           </RadioGroup>
         ) : (
-          <Skeleton />
+          <div className={classes.loader}>
+            <CircularProgress />
+          </div>
```

Comment:
I got confirmation from @pwgryglak and there should always be skeletons in all places where loading occurs